js/src/dom/eventHandler.js

```diff
@@ -0,0 +1,330 @@
+import Polyfill from './polyfill'
+import Util from '../util'
+
+/**
+ * --------------------------------------------------------------------------
+ * Bootstrap (v4.3.1): dom/eventHandler.js
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ * --------------------------------------------------------------------------
+ */
+
+/**
+ * ------------------------------------------------------------------------
+ * Constants
+ * ------------------------------------------------------------------------
+ */
+
+const namespaceRegex = /[^.]*(?=\..*)\.|.*/
+const stripNameRegex = /\..*/
+const keyEventRegex  = /^key/
+const stripUidRegex  = /::\d+$/
+const eventRegistry  = {}   // Events storage
+let uidEvent         = 1
+const customEvents   = {
+  mouseenter: 'mouseover',
+  mouseleave: 'mouseout'
+}
+const nativeEvents   = [
```

Comment:
Is this list of every event the browser supports really needed for Bootstrap?

If it's just a case of checking if an event is natively supported, maybe something a simple as this would suffice:

```js
function isNative(eventName) {
  return 'on' + eventName in document.documentElement;
}
```